util/fipstools/acvp/acvptool/testmodulewrapper/testmodulewrapper.go

```diff
@@ -90,39 +90,6 @@ func getConfig(args [][]byte) error {
 		"tweakMode": [
 		  "number"
 		]
-	}, {
-		"algorithm": "KAS-KDF",
-		"mode": "TwoStep",
-		"revision": "Sp800-56Cr2",
-		"capabilities": [{
-			"macSaltMethods": [
-				"random",
-				"default"
-			],
-			"fixedInfoPattern": "uPartyInfo||vPartyInfo",
-			"encoding": [
-				"concatenation"
-			],
-			"kdfMode": "feedback",
-			"macMode": [
-				"HMAC-SHA2-256"
-			],
-			"supportedLengths": [{
-				"min": 128,
-				"max": 512,
-				"increment": 64
-			}],
-			"fixedDataOrder": [
-				"after fixed data"
-			],
-			"counterLength": [
-				8
-			],
-			"requiresEmptyIv": true,
-			"supportsEmptyIv": true
-		}],
-		"l": 256,
-		"z": [256, 384]
```

Comment:
Why we don't have the entry for? 
```
"algorithm": "KDA",
"mode": "HKDF",
```
See
https://github.com/google/boringssl/commit/726585c58ec89f65ab669187665232790cbf3d02#diff-6fae1d42d6f42e954a91feb6e08d8802c80eb49863f45a76ed5d1ae84414398cR109

Reply:
We do, it's just not in the `testmodulewrapper.go` file since that file isn't used for testing our crypto implementation (It's just a golang implementation of an algorithm to ensure the parser works). You can find the entry you want [here](https://github.com/awslabs/aws-lc/blob/f6f2eb0f94add6cfecbbe528b87009b8a658e86c/util/fipstools/acvp/modulewrapper/modulewrapper.cc#L921).

Reply:
Thanks, Bill. Do we want to add this feature to `testmodulewrapper.go` or we have another way to ensure the parser works for hkdf?

Reply:
No need! the ACVP tool's offline tests test the parser (and our implementation) for both `testmodulewrapper.go` and `modulewrapper.cc`, so we're already ensuring that the parser for HKDF (and all the other algorithms in our ACVP tool) are being tested offline. You can find the file telling us what to check [here](https://github.com/awslabs/aws-lc/blob/main/util/fipstools/acvp/acvptool/test/tests.json).